docs/source/glossary.md

```diff
@@ -190,4 +190,141 @@ The Network Map Service accepts digitally signed documents describing network ro
+
+
+## Hyperledger-Indy
+
+This section lists specific terms used in Hyperledger-Indy.
+
+### Admin DID
+A decentralized identifier for Admin as defined by the DID Data Model and Generic Syntax specification.
+
+###  Admin Seed
+Seed can be any randomly chosen 32 byte value. It is not having a proper format and used to initializing keys. The seed used for Admin key is called an admin seed.
+
+### Agency
+ A service provider that hosts Cloud Agents and may provision Edge Agents on behalf of a Ledger’s Entities.
+
+### Agent
+A software program or process used by or acting on behalf of a Ledger’s Entity to interact with other Agents or, via a Ledger’s Client component, directly with the Ledger. Agents are of two types: Edge Agents run at the edge of the network on a local device, while Cloud Agents run remotely on a server or cloud hosting service. Agents typically have access to a Wallet in order to perform cryptographic operations on behalf of the Ledger’s Entity they represent.
+
+### Dependent
+An Individual who needs to depend on a Guardian to administer the Individual’s Ledger Identities. Under a Trust Framework, all Dependents may have the right to become Independents. Mutually exclusive with Independent.
+
+### Developer
+An Identity Owner that has legal accountability (in a scenario where there is a Trust Framework) for the functionality of an Agent, or for software that interacts with an Agent or the Ledger, to provide services to a Ledger Entity.
+
+### DID
+A decentralized identifier as defined by the DID Data Model and Generic Syntax specification. DIDs enable interoperable decentralized self-sovereign identity management. An Identity Record is associated with exactly one DID. A DID is associated with exactly one DDO.
+
+### Domain Genesis
+Domain genesis is a genesis file used to initialise the network and may populate network with some domain data.
+
+### Endorser
+Endorser is having the required rights to write on a ledger. Endorser submits a transaction on behalf of the original author.
+
+### Genesis Record
+The first Identity Record written to the Ledger that describes a new Ledger Entity. For a Steward, the Genesis Record must be written by a Trustee. For an Independent Identity Owner, the Genesis Record must be written by a Trust Anchor. For a Dependent Identity Owner, the Genesis Record must be written by a Guardian.
+
+### Identity
+A set of Identity Records, Claims, and Proofs that describes a Ledger Entity. To protect privacy: a) an Identity Owner may have more than one Ledger Identity, and b) only the Identity Owner and the Relying Party(s) with whom an Identity is shared knows the specific set of Identity Records, Claims, and Proofs that comprise that particular Identity.
+
+### Identity Owner
+A Ledger Entity who can be held legally accountable. An Identity Owner must be either an Individual or an Organization. Mutually exclusive with Thing.
```

Comment:
The last sentence "Mutually exclusive with Thing" is not clear, add more details. Also mention Dependent and Independent Identity owners as that has been mentioned above.